swift_kernel.py

```diff
@@ -503,4 +503,7 @@ def do_complete(self, code, cursor_pos):
 
 if __name__ == '__main__':
     from ipykernel.kernelapp import IPKernelApp
-    IPKernelApp.launch_instance(kernel_class=SwiftKernel)
+    # We pass the kernel name as a command-line arg, since Jupyter gives those
+    # highest priority (in particular overriding any system-wide config).
+    IPKernelApp.launch_instance(
+        argv=['--IPKernelApp.kernel_class=__main__.SwiftKernel'])
```

Comment:
When I try to start this kernel in a notebook, jupyter gives me a "Timeout waiting for kernel_info reply from be858647-d4ba-4ccf-8332-2bf5ca0ad3dd". I think it's because `IPKernelApp.launch_instance` isn't getting the commandline arguments that jupyter passes it for connection information. Changing this line to `argv=sys.argv + ['--IPKernelApp.kernel_class=__main__.SwiftKernel']` fixes the problem for me. Does that sound reasonable to you?

Reply:
Yes, that is indeed a bug in my one new line of code. /facepalm

Tried this updated version live, and it's working for me. (LMK if there are tests I should run beyond exercising it by hand.)

Reply:
I'm working on making the tests automated and easy to run -- in the meantime I'm manually running them myself on all the PRs. Your PR passes now :)

(If you do ever want to run tests yourself, `gcloud builds submit --config docker/cloudbuild.yaml .` or `cloud-build-local --config=docker/cloudbuild.yaml --dryrun=false .` should do it.)